src/reducer.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * This action type will be dispatched when your history
+ * receives a location change.
+ */
+export const LOCATION_CHANGE = '@@router/LOCATION_CHANGE'
```

Comment:
I'd like to leave the UPDATE_LOCATION action in place to reduce some of the upgrade burden on those who will continue to use the middleware. This one can be renamed to whatever we like.